cloudcomposerdiff/README.md

```diff
@@ -0,0 +1,184 @@
+# A python command line tool to diff two Cloud composer environments.
+
+## Running this code
+
+### Clone repository
+
+Clone the code from GitHub to your local machine
+
+### Install locally with pip
+
+```
+pip install cloudcomposerdiff
+```
+
+### Authorizing the application using gcloud
+
+Install [gcloud](https://cloud.google.com/sdk/gcloud)
+
+Select a user which has access to both composer environments to compare.
+
+Wit this user account, initialize gcloud:
+
+```
+gcloud init
+```
+
+Next, authorize the application to access cloud composer environments:
+
+```
+gcloud auth application-default login
+```
+
+### Use the appplication
+
+```
+cloudcomposerdiff \
+--env1_project_id <> \
+--env1_location <> \
+--env1_name <> \
+--env2_project_id <> \
+--env2_location <> \
+--env2_name <>
+```
+
```

Comment:
include an example command for users and what expected output would look like. If expected output is too verbose to put here, lmk and we can brainstorm what to do. 

Example of an example command
```shell
cloudcomposerdiff \
--env1_project_id my-project \
--env1_location us-central1 \
--env1_name my-environment-1 \
--env2_project_id my-project \
--env2_location us-east1 \
--env2_name my-environment-2
```

Reply:
As per my other comment, I have uploaded 3 videos (in the form of GIFs) which demonstrate using the tool. The videos show what the input looks likes and what the example output looks like (in tabular form). Please let me know if these videos are not clear & I need to add more details to the README